sass/routes/_about-modal.scss

```diff
@@ -14,7 +14,7 @@
                 align-items: center;
                 padding: 20px 25px;
                 border: none;
-                border-bottom: 1px solid #ddd;
+                border-bottom: 1px solid var(--center-channel-color-08);
```

Comment:
Can you use the rgba version of the theme colour here?

Reply:
what @hmhealey said! :D